### PR TITLE
feat: add option for reply to in context menu

### PR DESCRIPTION
**Pull request by scmmishra**
This PR adds reply to button in the context menu. 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

##### Enabled for Web Widget

CleanShot 2023-10-04 at 15 56 34

##### Disabled for Email

CleanShot 2023-10-04 at 15 57 37

##### Enabled for outgoing message in supported channel

CleanShot 2023-10-04 at 15 58 30

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] ~I have made corresponding changes to the documentation~
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/conversation/Message.vue [hunk 1]
```diff
@@ -269,6 +274,8 @@ export default {
         copy: this.hasText,
         delete: this.hasText || this.hasAttachments,
         cannedResponse: this.isOutgoing && this.hasText,
+        replyTo: false,
+        // replyTo: !this.data.private && this.inboxSupportsReplyTo,
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/Message.vue:278]: Let us use feature flag here. Otherwise looks fine.

Comment (scmmishra) [reply app/javascript/dashboard/components/widgets/conversation/Message.vue:278]: Noted, any reference to how I can add that. Couldn't find any specific utility except for `featureFlags.js` for holding the keys

Comment (scmmishra) [reply app/javascript/dashboard/components/widgets/conversation/Message.vue:278]: Should I be adding it to `features.yml`?

Comment (pranavrajs) [reply app/javascript/dashboard/components/widgets/conversation/Message.vue:278]: Let us pair.

**Review by muhsin-k — APPROVED** [review]

**Comment by nagorsky** [comment]
Is reply feature available for API channel ?

**Pull request merged**